### ISSUE TITLE
NVDA doesn't perceive some objects and groups in PowerPoint Slideshow

### ISSUE THREAD

**Issue by KarinInWinnipeg** [nvaccess/nvda#16161]
I'm having issues with NVDA reading a PowerPoint slide in presentation mode. I've attached a picture of the slide I made to test. It contains the objects in the following list, in that order. I opened the slideshow and 1) let NVDA read the slide "by itself" with no interaction, and then 2) used the down arrow key to read the slide. Both methods produced the same results. Pressing Tab/Shift Tab had no results. The issues with each object or group on the slide are in the following list after the object.

1. The title of the slide, which is the Title placeholder with text in it. This is read every time without issue.
2. A text box that is grouped with two shapes, a star and an arrow. Both shapes are marked as decorative. The text in the text box is read without issue, but the alt text for the group is ignored.
3. A grouped object containing a star shape and an arrow shape. The group has alt text and the shapes don't (but they do not have "mark as decorative"). The group and the shapes are ignored, and the alt text for the group is not read.
4. A star shape with alt text. This shape is ignored.
5. An arrow shape with alt text. The alt text for this shape is read.
6. A SmartArt shape with alt text and text inside the shape. The alt text for the shape is read, but the text inside the shape is ignored.

I tested this slide with JAWS also, and it recognized all the grouped and individual objects. It didn't read the text inside the text box that is grouped with the two graphics. It read the alt text for all the groups, the star, the arrow, and the SmartArt. It also read the text inside the SmartArt. I think the way that JAWS reads this slide is the expected behaviour.

Thanks for your attention to this.
![example of slide](https://github.com/nvaccess/nvda/assets/43084499/bdd63790-7447-4c01-9f12-1f10802aadbb)

**Comment by seanbudd** [comment]
Welcome @KarinInWinnipeg ,

We are unable to process this issue as it stands.

- Please open a new issue and fill out the bug issue template fully.
- To be able to reproduce this issue, we need a minimal sample. Please upload an example powerpoint file. You may need to put it in a zip folder first, or upload to Dropbox/Google cloud and provide a link.

Kind Regards,
NV Access Software Developers
